src/src/tx_main.cpp

```diff
@@ -1181,8 +1185,17 @@ void loop()
   if (TelemetryReceiver.HasFinishedData())
   {
       crsf.sendTelemetryToTX(CRSFinBuffer);
+#ifdef HAS_BLET
+      BluetoothTelemetryUpdateValues(CRSFinBuffer);
+#endif
       TelemetryReceiver.Unlock();
   }
+  else
+  {
+#ifdef HAS_BLET
+      BluetoothTelemetryUpdateValues(nullptr);
+#endif
+  }
```

Comment:
This should be removed and the null data case should be called via the timeout() in the event subsystem.